spec/mercure.md

```diff
@@ -36,10 +36,10 @@ Mercure is a protocol allowing to push data updates to web browsers and other HT
 The keywords **MUST**, **MUST NOT**, **REQUIRED**, **SHALL**, **SHALL NOT**, **SHOULD**, **SHOULD NOT**, **RECOMMENDED**, **MAY**, and **OPTIONAL**, when they appear in this document, are to be interpreted as described in [@!RFC2119].
 
 * Topic: An HTTP [@!RFC7230] or HTTPS [@!RFC2818] topic URL. The unit to which one can subscribe to changes.
-* Publisher: An owner of a topic. Notifies the hub when the topic feed has been updated. As in almost all pubsub systems, the publisher is unaware of the subscribers, if any. Other pubsub systems might call the publisher the "source". Typically a website or a web API, but can also be a web browser.
+* Publisher: An owner of a topic. Notifies the hub when the topic feed has been updated. As in almost all Pub/Sub systems, the publisher is unaware of the subscribers, if any. Other Pub/Sub systems might call the publisher the "source". Typically a website or a web API, but can also be a web browser.
```

Comment:
This part has been borrowed to https://www.w3.org/TR/websub/. Maybe can we keep it in sync?

Reply:
Sure.